src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -310,6 +356,160 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+#if UNIX
+        /// <summary>
+        /// Gets the identity name to use for writing to syslog.
+        /// </summary>
+        /// <returns>
+        /// The string identity to use for writing to syslog.
+        /// <para>
+        /// The default value is 'powershell'
+        /// </para>
+        /// </returns>
+        internal override string GetSysLogIdentity()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string identity = ReadValueFromFile<string>(fileName, "LogIdentity");
```

Comment:
Just curious, why don't we want the identity name to always be "powershell" or "pwsh"?

Reply:
Overriding the name allows side by side instances to be distinguished from each other in the log.

Reply:
Make sense. Thanks!